wave/src/Http/Controllers/WebhookController copy.php

```diff
@@ -0,0 +1,17 @@
+<?php
+
+namespace Wave\Http\Controllers;
+
+use \App\Http\Requests;
+use Illuminate\Http\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Laravel\Cashier\Http\Controllers\WebhookController as StripeWebhookController;
+
+class WebhookController extends StripeWebhookController
+{
+
+    public function handle(Request $request) {
+        $this->handleWebhook($request);
```

Comment:
This file can be removed, no need to keep a copy around if the original was changed to the same content.

Reply:
Ups, yes!